client/signup/jetpack-connect/index.jsx

```diff
@@ -136,8 +131,17 @@ class JetpackConnectMain extends Component {
 			this.props.jetpackConnectSite.isFetching;
 	}
 
-	getCurrentUrl() {
-		let url = this.refs.siteUrlInputRef.state.value.toLowerCase();
+	handleUrlChange = ( event ) => {
+		const url = event.target.value;
+		this.setState( {
+			currentUrl: this.cleanUrl( url ),
+			shownUrl: url,
+		} );
+		this.dismissUrl();
```

Comment:
Yes, that totally makes sense! I'm not sure that this wasn't done intentionally for some reason we don't understand though, @johnHackworth might have an idea?